python/tvm/driver/tvmc/compiler.py

```diff
@@ -185,11 +185,11 @@ def compile_model(
         with autotvm.apply_history_best(tuning_records):
             with tvm.transform.PassContext(opt_level=3):
                 logger.debug("building relay graph with tuning records")
-                graph_module = relay.build(mod, tvm_target, params=params, target_host=tvm_target)
+                graph_module = relay.build(mod, tvm_target, params=params, target_host=target_host)
```

Comment:
This looks unrelated to the overall CL addition, I guess as it is a minor variable replacement fix, we can leave it here - I'm just pointing it out.

Reply:
@leandron please be aware that we should include this fix in the back port, but we will not include this PR since this is a new feature. Accordingly, I'd suggest separating this fix.